cypress-tests/cypress/support/pageBuilder/pbDeleteAllMenus.ts

```diff
@@ -0,0 +1,88 @@
+import { gqlClient } from "../utils";
+
+const LIST_MENUS_QUERY = /* GraphQL */ `
+    query listMenus {
+        pageBuilder {
+            listMenus {
+                data {
+                    title
+                    slug
+                    description
+                    items
+                    createdOn
+                    createdBy {
+                        id
+                        displayName
+                        __typename
```

Comment:
Just in case - make sure `__typename` is removed on all of these queries that this PR touched.